week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,16 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var conversion = price * 1.4;
+
+  return conversion + conversion * 0.01;
```

Comment:
If the conversion fee changes from 0.01 to 0.05 for example, you would need to change that in two places. Can you think of a way to write your program so that `0.01` is only used once?

Reply:
var fee = 0.01;
function convertToUSD(price) {
  var conversion = price * 1.4;

  return conversion + conversion * fee;
}
function convertToBRL(price) {
  var conversion = price * 4.82;

  return conversion + conversion * fee;
}

Reply:
I defined a variable for 0.01 as a fee